.jenkins/build_docker_images.sh

```diff
@@ -1,23 +1,25 @@
 #!/bin/bash
 
+# Read in tag name used for the Docker images to be tested
+tagname=$1
+
 # Set variable to allow parallel building in the Docker image creation
 export DOCKER_BUILDKIT=1
 export BUILDKIT_PROGRESS=plain
-targets="build build_serialize_gt4py_dev"
-tags="hpc hpc-serialize"
 
 # Speed-up the compilations by using pre-built MPI, FMS, and ESMF images
 export BUILD_FROM_INTERMEDIATE=y
 
 # Build FV3 without and with Serialbox support enabled
 make pull_deps
-COMPILED_TAG_NAME=hpc make ${targets} 
+make DEP_TAG_NAME=$tagname build build_serialize 
```

Comment:
Is it intended that this changed from `build_serialize_gt4py_dev` to `build_serialize`? Just checking.

Reply:
I didn't know which of serialize or serialize_gt4py_dev to use.  Went with serialize, we can change too serialize_gt4py_dev in a separate PR